assets/src/components/map/tooltip.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { Tooltip } from "react-leaflet"
+
+interface MapTooltipProps {
+  title: string
+  permanent?: boolean
+}
+
+export const MapTooltip = ({ title, permanent }: MapTooltipProps) => {
+  return (
+    <Tooltip
+      key={`tooltip-permanent-${permanent}`}
```

Comment:
question: is this `key` prop needed?

Reply:
Sooooort of not really?

`<Tooltip />` doesn't re-render properly because Leaflet and React aren't the best of friends, so giving this a key to force a re-render is necessary for that reason.

BUT.... that's not really a use case I see us using - a tooltip will either be permanent or not.

Reply:
Oooh I see, that makes sense, I wanna think about it some more but I like this solution so far.

Reply:
at the very least, we probably wanna document this